packages/ui/src/components/Button/button.style.ts

```diff
@@ -6,60 +6,53 @@ export const ButtonStyles = styled.div`
   button:disabled .component-child {
     cursor: not-allowed !important;
   }
+  @import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;500;600;700&display=swap');
+
   button {
-    font-size: 1.6rem;
-    display: flex;
+    height: 40px;
+    width: 140px;
+    min-width: auto;
+    max-width: calc(50vw - 36px);
+    display: inline-flex;
     align-items: center;
-    justify-content: space-evenly;
-    cursor: pointer;
-    font-weight: 400;
-    line-height: 1.9rem;
-    transition: background-color ease 0.2s;
-    height: 50px;
-    min-width: 200px;
-    box-shadow: rgba(0, 0, 0, 0.12) 0px 5px 10px 0px;
-    padding: 0;
-    border-radius: 5px;
+    justify-content: center;
+    font-size: 14px;
+    font-weight: 500;
+    font-family: 'Inter', --apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
```

Comment:
I think we would like to have all our website having the same font. On how to do check this out
https://github.com/paschalidi/web-components/tree/master/src/GlobalStyles